src/test/java/com/fasterxml/jackson/databind/deser/creators/DelegatingCreatorImplicitNames2543Test.java

```diff
@@ -0,0 +1,74 @@
+package com.fasterxml.jackson.databind.deser.creators;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.introspect.AnnotatedMember;
+import com.fasterxml.jackson.databind.introspect.AnnotatedMethod;
+import com.fasterxml.jackson.databind.introspect.AnnotatedParameter;
+import com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector;
+import org.junit.Test;
+
+import static com.fasterxml.jackson.annotation.JsonCreator.Mode.DELEGATING;
+import static com.fasterxml.jackson.annotation.JsonCreator.Mode.PROPERTIES;
+import static com.fasterxml.jackson.databind.testutil.DatabindTestUtil.a2q;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DelegatingCreatorImplicitNames2543Test {
+    static class Data {
+
+        final String part1;
+        final String part2;
+
+        // this creator is considered a source of settable bean properties,
+        // used during deserialization
+        @JsonCreator(mode = PROPERTIES)
+        public Data(@JsonProperty("part1") String part1,
+                    @JsonProperty("part2") String part2) {
+            this.part1 = part1;
+            this.part2 = part2;
+        }
+
+        // no properties should be collected from this creator,
+        // even though it has an argument with an implicit name
+        @JsonCreator(mode = DELEGATING)
+        public static Data fromFullData(String fullData) {
+            String[] parts = fullData.split("\\s+", 2);
+            return new Data(parts[0], parts[1]);
+        }
+    }
+
+    private static class DelegatingCreatorNamedArgumentIntrospector
+            extends JacksonAnnotationIntrospector {
+        public String findImplicitPropertyName(AnnotatedMember member) {
+            if (member instanceof AnnotatedParameter p) {
+                if (p.getOwner() instanceof AnnotatedMethod m) {
```

Comment:
Cannot use this style of cast pre-JDK-17; Jackson has JDK 8 as baseline, so need to change to old style

Reply:
Yeah, I just noticed this too. Will update.

Reply:
np, CI was not running without my approval (since it's your 1st contribution), harder to notice.
Thanks for fixing it!